### PR TITLE
Added 0713 Lemon color to Isacord polyester

### PR DESCRIPTION
**Pull request by rayojohis**
Hi there, This is my first time contributing. I just want to make sure I am adding the color codes correctly.

I added only one as a test,  please let me know if the format works.

If it does, I have many more that I can add too.

Please note I am not a programmer, so any tips would be appreciated.

Thanks,

### CODE REVIEW

**Comment by kaalleen** [comment]
Looks good to me!

If you want, we can add you as a contributor to this repository. If you then create a new branch named `rayojohis/branch_name` and upload your changes, it will build a developer release version which everybody who is interested can test right away.

**Comment by rayojohis** [comment]
Hi, That would be great. Please add me as a contributor. I will add the rest of the colors I have now.

**Comment by kaalleen** [comment]
Ok great, I've send an invitation to you.

**Comment by nlighteneddesign** [comment]
@kaalleen Good morning! Is there a specific reason these files are fixed width columns?
Is there any concern with converting to a more standard CSV so it's easier to modify going forward?
It also specifies 4 columns at the top, but I think there are only 3 columns, is there a blank column to create the extra space?

I'm helping @rayojohis get her local environment set up and working on some tooling so she can more easily add new colors and deduplicate.

**Comment by nlighteneddesign** [comment]
Ok looking at this: https://testing.developer.gimp.org/core/standards/gpl/
It looks like the 4 columns are rgb, then the name, so the fixed width isn't likely important at all. Can you please confirm?

**Comment by kaalleen** [comment]
I'm not exactly sure how you want to format it. Can you post an example?

**Comment by kaalleen** [comment]
It looks as if you want to edit the file on the "code level". So this may not fit to your workflow, but there are two extensions that may help to insert new colors in a more graphical way. Palette to text and generate palette.

https://inkstitch.org/docs/thread-color/

**Comment by nlighteneddesign** [comment]
Ah, sorry I was not clear, the color .gpl file spec says it just needs four columns, R, G, B, and Name

The isacord number is part of the name and not a column on it's own so it's not a fixed width file, that is just for viewing.

@rayojohis has the contributor status now so we can test, but we'll reorder and add the new colors so we can confirm what I was reviewing.

Thanks!

**Comment by kaalleen** [comment]
Oh I see. Actually that is exactly how we parse it. Split into three columns and get the catalog number from the third entry. So I am afraid you have to stay with the current format.

You don't really have to build the whole thing to test color palettes. You can just restart inkscape. It will load the palette files from the inkscape palette folder. So you can adapt files there and copy it into the inkstitch palette folder to finally upload it to inkstitch.

**Comment by nlighteneddesign** [comment]
Great! We'll test that way

**Comment by rayojohis** [comment]
This is ready for review. I tested locally. I order everything by Isacord number, so it is easy to find.

One thing I noticed is that locally I had to update the Inkscape Palette, as well as the Inkstitch Palette. Do I need to create a pull request for Inkscape as well? or Is that file a cache from Inkstitch?

**Comment by kaalleen** [comment]
The Ink/Stitch extension `Install thread color palettes for Inkscape` will copy the Ink/Stitch palettes to the Inkscape palette folder so they are available to use. No need to make a merge request for the Inkscape repo itself as these color palettes are Ink/Stitch specific.

**Review by kaalleen — APPROVED** [review]

File: palettes/InkStitch Isacord Polyester.gpl [hunk 1]
```diff
@@ -2,394 +2,402 @@ GIMP Palette
...
+ 153 190 51	       	             Limabean	  5833
+    90 60 0	       	           Herb Green	  5866
+  158 196 0	       	           Erin Green	  5912
+ 140 160 90	       	          Grasshopper	  5933
+ 153 153 51	       	           Moss Green	  5934
+ 217 255 66	       	      Sour Apple neon	  5940
+  80 110 51	       	       Backyard Green	  5944
+ 255 255 63	       	    Mountain Dew neon	  6010
+190 231 110	       	             Tamarack	  6011
+ 218 224 86		                  Limelight	  6031
```
Comment (ricardoquesada) [inline palettes/InkStitch Isacord Polyester.gpl:396]: wrong identation

**Comment by kaalleen** [comment]
Just an information about the columns specification at the top in the palette files.
They don't define the columns in the file itself, but define the "line breaks" within the inkscape color swaps. So when you change the size of the swaps in Inkscape, that is how they will stick together. So you can ensure that for example all the reddish colors will be displayed next to each other without a line break. How can I describe it best? It is for grouping the colors.

File: palettes/InkStitch Isacord Polyester.gpl (hunk 1 above)
Comment (nlighteneddesign) [reply palettes/InkStitch Isacord Polyester.gpl:396]: Yeah it turns out the indentation doesn't actually matter too much, it should still work.

We did our best to line them up, but there were some edge cases where it didn't align.

**Comment by nlighteneddesign** [comment]
> Just an information about the columns specification at the top in the palette files. They don't define the columns in the file itself, but define the "line breaks" within the inkscape color swaps. So when you change the size of the swaps in Inkscape, that is how they will stick together. So you can ensure that for example all the reddish colors will be displayed next to each other without a line break. How can I describe it best? It is for grouping the colors.

Oh that makes so much more sense!

**Pull request merged**